### ISSUE TITLE
Does nexus supports arm64

### ISSUE THREAD

**Issue by Abhilash-sandupatla** [sonatype-nexus-community/nexus-repository-composer#130]
Can we install nexus on arm64 node ?

**Comment by TuningYourCode** [comment]
Yes you can but there are no finished container etc. as far as i know.

For development of the nexus-repository-composer plugin we build it on MacBooks with arm64 cpus. See https://github.com/FocusedDevs/nexus-repository-composer?tab=readme-ov-file#build-with-docker and https://github.com/FocusedDevs/nexus-repository-composer/blob/master/Dockerfile

if you don't need the composer plugin have a look at the upstream [Dockerfile](https://github.com/sonatype/docker-nexus3/blob/main/Dockerfile.java17)
